leapp/repository/actor_definition.py

```diff
@@ -175,6 +175,7 @@ def serialize(self):
             'class_name': self.class_name,
             'description': self.description,
             'tags': self.tags,
+            'config_schemas': self.config_schemas,
```

Comment:
isn't  ```'configs': self.configs,``` missing here?

Reply:
I thought that serialize would need to be used for things that are class attributes rather than the directories but I'll give that a try.

Reply:
I was just going through the PR and tried to see if I can find what's missing and just stumbled upon this.
From what I can tell just by looking at the PR, it does seem like you did it right. I will get a snapshot of your PR and try it and have a look if I can find the issue.

Btw this is feature that asks for adding snactor support.

Reply:
> Btw this is a feature, that asks for adding snactor support.

You are right, however we decided to have updated snactor as a stretch goad for the next (spring) release and possibly finish it in the other one.